components/PrimaryLayout.js

```diff
@@ -8,13 +8,13 @@ import Navbar from './Navbar';
 
 export default function PrimaryLayout({ children }) {
   return (
-    <>
+    <div className="flex flex-col min-h-screen">
```

Comment:
Awesome @Sahil-101! This works like a charm 🔮  The only thing I'd say is it might be better if this div with the Tailwind classes here be moved to line 15 just above the `NavBar`. Ideally the head info should be outside of the layout elements. You'll need to re-add the <> </> for this to work though as React needs fragments for multiple elements.

Reply:
Oh would change it, I understood Thanks for pointing out :sweat_smile:

Reply:
@Sahil-101 this is perfect!